ShoppingCart/CartHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShoppingCart
+{
+    public class CartHandler
+    {
+        private Cart _cart;
+                
+        public CartHandler(Cart cart)
+        {
+            _cart = cart;
+            _cart.CartStatus = CartStatus.Active;
+        }
+
+        public bool CartContainsItem(CartItem cartItem)
```

Comment:
This should be private method

Reply:
After removing a product completely, such product should not exist. So for testing purpose I had kept this method public, Please suggest what to do in such a scenario

Reply:
Nobody is using this method from outside class so no need to make it public

Reply:
I had kept this method public solely for testing the removal of products. Please suggest what could be more suitable by keeping this method private